public/give.html

```diff
@@ -20,8 +20,8 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.i18n/1.0.7/jquery.i18n.emitter.min.js"></script>
     <script src="i18n.js"></script>
 
-    <script defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDSz0lnzPJIFeWM7SpSARHmV-snwrAXd2s&libraries=geometry"></script>
-    <script type="module" src="locations-list-map.js"></script>
+    <!-- NOTE: Maps initialized dynamically in this script (if needed). Use "mapsKey" query string param to define your own key. -->
+    <script defer type="module" src="locations-list-map.js"></script>
```

Comment:
I probably shouldn’t have had this `defer` here...

Reply:
Actually gonna go ahead and remove this; this is a [local script anyway](https://www.w3schools.com/tags/att_script_defer.asp).